WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
+    @Environment(\.presentationMode) var presentation
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+
+                /// Custom Packages
+                ///
+                if viewModel.customPackages.count > 0 {
+                    ListHeaderView(text: Localization.customPackageHeader.uppercased(), alignment: .left)
+                        .background(Color(.listBackground))
+                }
+                ForEach(viewModel.customPackages, id: \.title) { package in
```

Comment:
Can two packages have the same name? Could this `id` become problematic at some point?

Reply:
In custom packages, the title is the id (it is therefore unique). I have added a comment in the networking model that best describes this thing. 663444b4f4d3b2693b0657a55807d4de511d42a4